main.tf

```diff
@@ -20,8 +20,13 @@ module "obs_staging_vpc" {
   create_iam_instance_profile_ssm_policy = "true"
   deploy_ec2_instance_db                 = false
   deploy_rds_db                          = true
+  xchain_settings = {
+    enabled      = true
+    docker_image = "omniops/xchain-indexer:latest"
+    config       = "staging"
+  }
   blockscout_settings = {
-    blockscout_docker_image = local.blockscout_docker_image
+    blockscout_docker_image = "omniops/blockscout:xchain"
```

Comment:
using custom blockscout image for staging rn as we cannot merge the blockscout ui pr yet or else that might deploy to testnet-1 also.

Reply:
can we set staging to use the latest, but set a static version for testnet? this will be important in the future to so we can push frequent changes without worrying about breaking testnet

Reply:
We can update it later once the blockscout PR is merged and once we have stress tested this staging changes properly

Reply:
Updated it in https://github.com/omni-network/blockscout-terraform/pull/4